index.js

```diff
@@ -36,7 +36,10 @@ function resolveUrlLoader(content, sourceMap) {
 
   // webpack 1: prefer loader query, else options object
   // webpack 2; prefer loader options
-  var options = defaults(loaderUtils.getOptions(loader), loader.options[camelcase(PACKAGE_NAME)], {
+  var options = defaults(
+      loaderUtils.getOptions(loader), 
+      loader.options && loader.options[camelcase(PACKAGE_NAME)], 
+      loader.query, {
```

Comment:
@acoll It seems to me that `loader.query` is redundant given that `loaderUtils.getOptions(loader)` incorporates it [here](https://github.com/webpack/loader-utils#getoptions).